### DISCUSSION TITLE
Can I disable text-overflow: ellipsis on TableView's Cell?

### DISCUSSION THREAD

**Discussion by mathieu-lessard** [adobe/react-spectrum#4418] [Q&A] [answered]
I have an input in a table, and when resizing I occasionally can see an ellipsis.

https://codesandbox.io/s/react-spectrum-table-ellipsis-6p63m1?file=/src/App.tsx

![screenshot_252](https://user-images.githubusercontent.com/2773112/233472996-ad460a33-c772-4141-bb69-790bc7305415.png)

Is there a way to disable the ellipsis?

Thanks!

**Comment by snowystinger** [comment, marked as answer]
Yes, you can render a div that's full size inside the Cell and just not turn on ellipsis for that.

On another note.
You're going to run into an issue if you put an input in a table currently. We haven't yet developed "edit mode". https://github.com/adobe/react-spectrum/issues/2328
